region/maridia/outer.json

```diff
@@ -1451,6 +1473,68 @@
                   "requires": []
                 }
               ]
+            },
+            {
+              "id": 5,
```

Comment:
These strats look hard without knowing what the next room is. Maybe we should add a tech for knowing what room is coming next?

Reply:
There are a bunch of ways to look at this problem.  A general readyForNextRoom tech that extends into readyHow's could be a start, for now.  The shinespark strat can be broken into expecting this room and not, with some different spark numbers.